core/src/main/java/org/apache/druid/common/guava/GuavaUtils.java

```diff
@@ -62,4 +100,70 @@ public static Long tryParseLong(@Nullable String string)
 
     return null;
   }
+
+  /**
+   * Try the various methods (zero arguments) against the object. This is handy for maintaining guava compatability
+   *
+   * @param object  The object to call the methods on
+   * @param methods The sequence of methods to call
+   * @param <T>     The return type
+   *
+   * @return The result of invoking the method on the object
+   */
+  private static <T> T tryMethods(Object object, Class<T> assignableTo, String... methods)
```

Comment:
Documentation should reference the specific commit and the range of lines (as a Github link) in Guava from where this was copied. Same for other non-trivial parts of the copied code.

Reply:
This part was not copied. The method names in the callers are copied though. I'll add a javadoc here to help make sure a "use the git tag as a reference" pattern is followed in the future for such behaviors.

Reply:
fixed